client/me/purchases/remove-purchase/index.jsx

```diff
@@ -123,10 +106,10 @@ const RemovePurchase = React.createClass( {
 		this.setState( { isDialogVisible: true } );
 	},
 
-	openChat() {
-		olarkActions.expandBox();
-		olarkActions.focusBox();
+	chatButtonClicked( event ) {
 		this.recordChatEvent( 'calypso_precancellation_chat_click' );
+		event.preventDefault();
```

Comment:
The reason I wouldn't put this in the `HappychatButton` is because its not necessary for the button to work and it has no meaning when looking at that code.